obs_maven/repo.py

```diff
@@ -59,8 +60,15 @@ def parse_primary(self):
         logging.debug("Parsing primary %s", primary_url)
         for cnt in range(1, 4):
             try:
-                with urllib.request.urlopen(primary_url) as primary_fd:
-                    with gzip.GzipFile(fileobj=primary_fd, mode="rb") as gzip_fd:
+                with tempfile.TemporaryFile() as tmp_file:
```

Comment:
Is the temporary file automatically removed?

Reply:
Yes, it's removed when it's closed by the context manager.